src/libPersistence/ContractStorage2.cpp

```diff
@@ -412,6 +412,9 @@ bool ContractStorage2::FetchContractFieldsMapDepth(const dev::h160& address,
 }
 
 void UnquoteString(string& input) {
+  if (input == "\"\"") {
```

Comment:
1. Software will crash if input is empty
2. What will happen if input is "\"     \""

Reply:
will be fine. if a space being there the key is not empty after unquote